presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestGrantRevoke.java

```diff
@@ -60,6 +60,8 @@ public void setup()
     public void testGrantRevoke()
     {
         //test GRANT
+        aliceExecutor.executeQuery(format("GRANT SELECT ON %s TO bob", tableName));
```

Comment:
No, not at this point (see the call to `assertAccessDeniedOnAllOperationsOnTable` that asserts that bob doesn't have any privilege on this table to begin with).